DESCRIPTION

```diff
@@ -15,13 +15,18 @@ License: Apache License 2.0 | file LICENSE
 URL: http://github.com/insightsengineering/rlistings
 BugReports: http://github.com/insightsengineering/rlistings/issues
 Depends: 
-    formatters (>= 0.3.2.5),
+    R (>= 4.0),
+    formatters (>= 0.3.2.5)
+Imports:
+    checkmate,
+    lifecycle,
```

Comment:
`lifecycle` should be used for documenting functions with their [status of development](https://lifecycle.r-lib.org/articles/stages.html) (so we can add the "experimental" tags/badges in the docs elegantly). I just followed the example of `teal` from CoreDev and put it in `Imports:`. I think this may be very useful to keep track of the status of each function. Moreover, the package has some nice functionalities like `deprecated_warn()` ([https://lifecycle.r-lib.org/reference/deprecate_soft.html](https://lifecycle.r-lib.org/reference/deprecate_soft.html))

Reply:
anyway I think you are right, I will try to put it in the suggested and remove the line with imports

Reply:
Done